runtime/jcl/common/reflecthelp.c

```diff
@@ -1720,3 +1720,233 @@ getFieldsHelper(JNIEnv *env, jobject cls)
 	vmFuncs->internalExitVMToJNI(vmThread);
 	return result;
 }
+
+/* Determine whether currentMethod is the accessor method for a record component with name componentName.
+ * an accessor method will have the same name as the component and take zero parameters.
+ */
+static U_8
+isRecordComponentAccessorMethodMatch(J9Method *currentMethod, const char* componentName, const char* componentSignature) {
```

Comment:
This would be more efficient if the componentName length was passed in as well as there is a potential for loop calling this and it would save doing strlen over and over again